README.md

```diff
@@ -115,6 +115,18 @@ Deploys the Conbench API server to a local minikube-powered Kubernetes cluster.
 This also deploys a kube-prometheus-based observability stack.
 Use this target for local development in this area.
 
+* `make docs-build`: Builds HTML docs locally so you may check that they render
+  correctly with no linting problems. You may install dependencies with
+  ```
+  pip install \
+      -r requirements-dev.txt \
+      -e ./benchclients/python \
+      -e ./benchalerts \
+      -e ./benchadapt/python \
+      -e ./benchconnect \
+      -e ./benchrun/python \
```

Comment:
We can keep this here, but we should probably also mention that installing _all the_ dependencies (eg benchalerts, etc.) isn't totally necessary if one isn't touching that. 

If I were contributing to the philosophy + general docs having to have functional installs of all of those is a pretty big hurdle to clear that isn't in principle important for us (right now at least).

Reply:
Yeah great point. Clarified!